src/main/webapp/2Dscript.js

```diff
@@ -658,12 +658,12 @@ function clearSelectedPaths(pathArray) {
  * Compute the subsection of a line contained in a bounding rectangle.
  * @param {Array<Point>} pose
  * @param {number} currentLat
- * @param {number} currentLng
  * @param {number} priorLat
+ * @param {number} currentLng
  * @param {number} priorLng
  * @return {subSectionData}
  */
-function computeSubSection(pose, currentLat, currentLng, priorLat, priorLng) {
```

Comment:
This was correct as-is. We prefer to keep arguments ordered as (lat, lng) pairs since the concept is so common.